nvflare/fuel/f3/communicator.py

```diff
@@ -0,0 +1,216 @@
+# Copyright (c) 2021-2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import os
+from typing import Optional
+
+from nvflare.fuel.f3 import drivers
+from nvflare.fuel.f3.comm_error import CommError
+from nvflare.fuel.f3.drivers.driver import Driver
+from nvflare.fuel.f3.drivers.driver_manager import DriverManager
+from nvflare.fuel.f3.drivers.driver_params import DriverParams
+from nvflare.fuel.f3.drivers.net_utils import parse_url
+from nvflare.fuel.f3.endpoint import Endpoint, EndpointMonitor
+from nvflare.fuel.f3.message import Message, MessageReceiver
+from nvflare.fuel.f3.sfm.conn_manager import ConnManager, Mode
```

Comment:
Could we use relative import for these?
For example `from .drivers.driver import Driver`

The reason is that, if later we decide to move this package into another folder or rename.
Say nvflare.f3 or nvflare.fuel.communication, then we don't need to change all these imports again.

Reply:
Good idea. Will try.